README.md

```diff
@@ -20,6 +20,8 @@ Please read the [Code of Conduct](CODE_OF_CONDUCT.md) and [guidelines](CONTRIBUT
 
 - **[Audere](https://www.auderenow.org/)** | _Health_ | Seattle, WA | **Remote friendly**? ✅ | [Current openings](https://www.auderenow.org/careers) | **Nonprofit**? ✅ | **Mission**: Improve global health outcomes in the world's most underserved communities; building products that enable people of any skill level to administer, interpret, and digitally record the world's most utilized rapid diagnostic tests
 
+- **[ChargePoint](https://www.chargepoint.com/)** | _Environmental_ | Campbell, CA; Amsterdam, Netherlands; Gurgoan, India | **Remote friendly**? ✅ | [Current openings](https://www.chargepoint.com/about/opportunities) | **Nonprofit**? ❌ | **Mission**: We're building one of the world’s leading EV charging networks across North America and Europe to facilitate the rapid transition to a sustainable electric future that is good for the planet, good for society and good for business.
```

Comment:
Change the mission to the company's mission statement: "to create technology that will enable the movement of all people and goods on electricity"

And I'll merge :)

Reply:
Yes, that's much snappier. Thanks @cherscarlett!